ReactiveCocoa/NSObject+RACPropertySubscribing.m

```diff
@@ -31,7 +31,7 @@ - (RACSignal *)rac_valuesForKeyPath:(NSString *)keyPath observer:(__weak NSObjec
 		setNameWithFormat:@"RACObserve(%@, %@)", self.rac_description, keyPath];
 }
 
-- (RACSignal *)rac_valuesAndChangesForKeyPath:(NSString *)keyPath options:(NSKeyValueObservingOptions)options observer:(__weak NSObject *)weakObserver {
```

Comment:
This changes the semantics of this method, so even though it might compile now, this is certainly not the right fix.
The error in #2761 says "in file using manual reference counting", but `ReactiveCocoa` only supports `RAC`.